pkg/report/testdata/openbsd/report/8

```diff
@@ -0,0 +1,3 @@
+TITLE: panic: kernel diagnostic assertion "pg->wire_count == 1" failed: file "/sys/kern/vfs_biomem.c", line 329
```

Comment:
I don't mind merging this as-is as we are moving in the right direction.
However, since we have syzbot, once we touch this, we get a splash of old reports with new titles. We change it again, we get another splash. Somebody will need to go and dup/invalidate old ones. We are not yet there for openbsd, but for linux we literally cannot touch this anymore for some types of reports even if we want (and we do want, but it would produce like half thousand duplicates flooding a thousand of people). So it can make sense to converge to final version sooner.
Line numbers don't work well. Any source change will cause duplicate bugs.
Too long titles cause problems, they don't fit into email subjects, poorly play with #syz dup commands (e.g. my email client can't send plain text email with line longer than 80 cols, so I literally incapable of duping long bugs).
What seems to work best is: bug-type in function-name
But at least I would suggest to shorten all constant parts: "panic: kernel diagnostic assertion ... failed: file ... line". The only information contained in all that text is basically that it's an assert. I would do something like "kernel assertion ... failed", or even "assert failed: ...". "file" may be replaced with "in". "line N" can be replaced with appending ":N" to file name (assuming that we leave it). Instead of:
```
panic: kernel diagnostic assertion "pg->wire_count == 1" failed: file "/sys/kern/vfs_biomem.c", line 329
```
it could be:
```
assert failed: "pg->wire_count == 1" in vfs_biomem.c
```
which looks as informative to me. Note that the full message is still available untouched in the log and report, so it's not that we are altering the primary info.